vueui/src/collections.js

```diff
@@ -0,0 +1,271 @@
+/**
+ * @file
+ * @copyright 2020 Aleksej Komarov
+ * @license MIT
```

Comment:
License is uncompatible with codebase licence (I think)

Reply:
Wrong. MIT can be incorporated into AGPL. AGPL cannot be incorporated into MIT.

Reply:
I've asked Skull and it seems MIT is fine with our license.